out.fil

```diff
@@ -0,0 +1,55 @@
+Generating api protos
```

Comment:
Mistakenly pushed debug output?

Reply:
oops...I created this to share with you on slack and forgot to erase before PR.  Fixed